src/lib/components/Treebank/Sentence/Sentence.js

```diff
@@ -31,6 +48,10 @@ const WrappedSentence = ({
     }
   }, [id, json]);
 
+  useEffect(() => {
```

Comment:
I moved this here (where it should have been) from `<Graph>`. What this does is reset the active word when the sentence changes.